src/CodiceFiscale/Calculator.php

```diff
@@ -83,7 +85,7 @@ private function calculateSurname()
     /**
      * Calculate the name part of the codice fiscale.
      *
-     * @returns Returns the name part of the codice fiscale.
+     * @return string Returns the name part of the codice fiscale.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.